src/MessageReader.js

```diff
@@ -81,7 +81,13 @@ class StandardTypeReader {
       this._intializeTextDecoder();
     }
     if (this._decoder) {
-      return this._decoder.decode(codePoints);
+      // TextDecoder does not support Uint8Arrays that are backed by SharedArrayBuffer, so copy the array here.
```

Comment:
Looks like it does work in NodeJS, but not in Firefox or Chrome. Maybe NodeJS is wrong. Anyway, seems like a good change -- broadens compatibility, and the overhead will be small (well-predicted branch, and there's a check above saying the string length is >=1000 chars.)